cmd/pebble/main.go

```diff
@@ -18,11 +18,22 @@ import (
 	"fmt"
 	"os"
 
+	"github.com/canonical/pebble/client"
 	"github.com/canonical/pebble/internals/cli"
 )
 
 func main() {
-	if err := cli.Run(nil); err != nil {
+	cliOptions := &cli.RunOptions{
+		ClientConfig: &client.Config{
+			Socket:        os.Getenv("PEBBLE_SOCKET"),
+			BaseURL:       os.Getenv("PEBBLE_BASEURL"),
+			BasicAuthUser: "fred",
+			BasicAuthPass: "gifappel",
```

Comment:
These two are just for testing purposes (to be removed), but can be used by derived projects that retrieves basic auth from a DB at this point.

Reply:
Removed.